_layouts/default.html

```diff
@@ -0,0 +1,81 @@
+<!--[if IE 9]>    <html class="no-js lt-ie10" lang="en" dir="ltr"> <![endif]-->
+<!--[if gt IE 8]><!-->
+<html class="js" lang="en" dir="ltr">
+<!--<![endif]-->
+
+<head>
+
+    <meta charset="UTF-8" />
+    <meta name="description" content="" />
+    <meta name="keywords" content="" />
+    <meta name="author" content="Canonical" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=8"><![endif]-->
+
+    <title>{{ page.title }}</title>
+
+    <!-- stylesheets -->
+    <link rel="stylesheet" type="text/css" media="screen" href="demo/ubuntu-styles.css" />
+    <!-- <link rel="stylesheet" type="text/css" media="screen" href="http://assets.ubuntu.com/sites/ubuntu/1284/u/css/beta/global-responsive.css" /> -->
+    <link rel="stylesheet" type="text/css" media="print" href="http://assets.ubuntu.com/sites/ubuntu/1284/u/css/core-print.css" />
+
+    <link rel="icon" href="favicon.ico" type="image/x-icon">
+
+    <!-- google fonts -->
+    <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,300,300italic,400italic,700,700italic%7CUbuntu+Mono' rel='stylesheet' type='text/css' />
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.1.1/gh-fork-ribbon.min.css" />
+<!--[if lt IE 9]>
+    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/github-fork-ribbon-css/0.1.1/gh-fork-ribbon.ie.min.css" />
+<![endif]-->
+    <style>
+    .github-fork-ribbon {
+        background-color: #dd4814;
+        font-family: Ubuntu;
+    }
+    </style>
+
+    <!-- javascript -->
+    <script src="http://assets.ubuntu.com/sites/ubuntu/1284/u/js/plugins/Imager.js"></script>
+</head>
+
+<body>
+    <div class="github-fork-ribbon-wrapper right">
+        <div class="github-fork-ribbon">
+            <a href="https://github.com/ubuntudesign/vanilla-framework">Fork me on GitHub</a>
+        </div>
+    </div>
+
+    <header class="banner global" role="banner">
+        <nav role="navigation" class="nav-primary nav-right">
+            <span id="main-navigation-link"><a href="#main-navigation">Jump to site nav</a></span>
+            <div class="logo">
+                <a class="logo-ubuntu" href="/">
+                    <img width="106" height="25" src="http://assets.ubuntu.com/sites/ubuntu/1284/u/img/logos/logo-ubuntu-orange.svg" alt="" />
+                    <span>Vanilla framework</span>
+                </a>
+            </div>
+        </nav>
+    </header>
+    <div class="wrapper">
+        <div id="main-content" class="inner-wrapper">
+            {{ content }}
+        </div>
+    </div>
+    <footer class="global clearfix no-global">
+        <div class="legal clearfix">
+            <p class="twelve-col">© 2015 Canonical Ltd. Ubuntu and Canonical are registered trademarks of Canonical Ltd.</p>
+            <p class="twelve-col">Code and documentation copyright 2015 Canonical Ltd. Licencing LGPLv3.</p>
```

Comment:
It may be, but I don't think that's how copyright should be presented. I'm not even sure what Copyright is supposed to mean when we have a license anyway. Can't we just say "Code licenced with LGPLv3 By Canonical Ltd." ?